examples/plot_simulate_lfp.py

```diff
@@ -0,0 +1,130 @@
+# to visualize the cells of the network to decide on the placement of each
+# electrode.
+net.plot_cells()
+
+###############################################################################
+# Electrode positions are stored under ``Network.lfp`` as a list
+# of tuples. Once we have chosen x,y,z coordinates for each electrode, we can
+# add them to the simulation.
+
+# XXX coordinates FUBAR, cortical depth direction is Y in NEURON objects!
+# electrode_pos = [(2, 400, 2), (6, 800, 6)]
+depths = list(range(-400, 1900, 200))
+electrode_pos = [(45, dep, 45) for dep in depths]
+net.add_electrode(electrode_pos)
+print(net.lfp)
+net.plot_cells()
+
+with MPIBackend(n_procs=4):
+    dpl = simulate_dipole(net)
+###############################################################################
+# Now that our electrodes are specified, we can run the simulation. The LFP
+# recordings are also stored under ``Network.lfp``.
+import matplotlib.pyplot as plt
+
+plt.figure()
+trial_idx = 0
+plt.plot(dpl[trial_idx].times, net.lfp[1]['data'][trial_idx])
+plt.plot(dpl[trial_idx].times, net.lfp[-2]['data'][trial_idx])
+plt.legend([f'e_pos {electrode_pos[1]}', f'e_pos {electrode_pos[-2]}'])
+plt.xlabel('Time (ms)')
+plt.ylabel(r'Potential ($\mu V$)')
+plt.show()
+
+from hnn_core.viz import _decimate_plot_data
+from hnn_core.dipole import _hammfilt
+from matplotlib.colors import SymLogNorm
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+
+import numpy as np
+window_len = 10  # ms
+decimate = [5, 4]  # from 40k to 8k to 2k
+winsz = np.round(1e-3 * window_len * dpl[trial_idx].sfreq)
+
+plot_data = list()
+for elec in net.lfp:
+    filt_data = _hammfilt(elec['data'][trial_idx], winsz)
+    decim_data, decim_time, lfp_sfreq = _decimate_plot_data(
+        decimate, filt_data, dpl[trial_idx].times, sfreq=dpl[trial_idx].sfreq)
+    plot_data.append(decim_data)
+plot_data = np.array(plot_data)
+
+fig, axs = plt.subplots(3, 1, sharex=True, figsize=(8, 8),
+                        gridspec_kw={'height_ratios': [2, 1, 2]})
+im = axs[0].pcolormesh(decim_time, depths, plot_data,
+                       norm=SymLogNorm(linthresh=1e-1, linscale=1.,
+                                       vmin=-10e2, vmax=10e2, base=10),
+                       cmap='BrBG_r', shading='auto')
+axs[0].set_ylabel(r'Distance from soma in Y ($\mu m$)')
+axs[0].set_xticks(np.arange(10, 250, 10), minor=True)
+axs[0].grid(True, which='both', axis='x')
+axins = inset_axes(axs[0],
+                   width="2%",  # width = 5% of parent_bbox width
+                   height="80%",  # height : 50%
+                   loc='lower left',
+                   bbox_to_anchor=(1.075, 0.1, 1, 1),
+                   bbox_transform=axs[0].transAxes,
+                   borderpad=0,
+                   )
+cbh = fig.colorbar(im, cax=axins, extend='both')
+cbh.ax.yaxis.set_ticks_position('left')
+cbh.ax.set_ylabel(r'Potential ($\mu V$)')
```

Comment:
This code is pretty sophisticated and advanced for most folks. Could we add `plot` method to `LFPElectrode` and put it there?

Reply:
Oh yeah :) I put it in to start a discussion on what we actually want to show. I'm actually not convinced---the spikes in our network generate unreasonably large LFP fluctuations, probably due to the extreme synchrony with which they fire. I'll create a much-simplified proposal and focus on writing (accurate) tests to keep us on the straight and narrow when we iterate on LFPs and CSDs in future PRs.